instagram-app/src/components/Login/Login.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+
+class Login extends React.Component {
+    constructor(){
+        super();
+        this.setState = {}
+    }
+    loggedIn = (e) => {
+        e.preventDefault();
+        const user = document.getElementById("user").value;
+        console.log(user.value)
+        localStorage.setItem('user', user)
+        window.location.reload()
```

Comment:
Since we are using React for this project, instead of using `document.getElementById`, it would be better practice to turn the `<input />` element holding the username text into a _controlled input_ with a state-based value and a corresponding `onChange` event method.